### DISCUSSION TITLE
Emulation Type of Belkin/Hue Emulation

### DISCUSSION THREAD

**Discussion by P0ll0ck** [arendst/Tasmota#14613] [Q&A] [answered]
Hey, im just trying to set up some shelly i3 using your amazing software to get rid of the non smart switches in my flat. im using an amazon echo plus as bridge and tried to integrate an i3 using tasmota with both, belkin and hue emulation activated. with hue, the detected device type within the echoes software is fixed to a lamp, using belkin mode im able to switch between lamp or plug but actually there is no option to use the i3 as a switch. is this a limitation of the tasmota software or of the implementation within the echos detecting routine and would it be possible to implement a switch emulation mode within the tasmota device? would be absolutely perfect for my (and im sure alot of other users) application, so i wont need any further hardware besides the shellys.

Greetings from Germany and thanks for all your efforts to support this amazing software (:

Edit: And im sorry if i just did not found the correct configuration option if it is already implemented

**Comment by s-hadinger** [comment, marked as answer]
This is a limitation from Alexa/Hue/Wemo emulation. It only supports bulb for Hue, and bulb/plug for Wemo. Any other type of device requires an Alexa skill.

**Reply by P0ll0ck** [reply]
Hey s-hadinger, thanks alot for the quick reply! can you tell me, which skill is needed for the other device types?

**Reply by s-hadinger** [reply]
I have done skill to send IR commands from Alexa via a Tasmotized IR blaster. But I have no experience of a switch that would trigger an Alexa routine. Maybe you can find some in the open-source world.

**Comment by NsinghP** [comment]
> I have done skill to send IR commands from Alexa via a Tasmotized IR blaster. But I have no experience of a switch that would trigger an Alexa routine. Maybe you can find some in the open-source world.

@s-hadinger How can I get this skill that you have done to send IR Commands from Alexa via Tasmotized Ir Blaser? Apologies for bumping up an old thread.

**Reply by s-hadinger** [reply]
I did to find it back, please give me a couple of days

**Reply by NsinghP** [reply]
Just a reminder for the alexa skill to send IR commands via Tasmotized IR blaster, I hope you find it @s-hadinger , Thanks.

**Reply by s-hadinger** [reply]
Still working on it...
